export const failureCallbackFunction = (error: any, inputVariable: any) => {
  void error;
  return inputVariable;
};
